### PR TITLE
fix ci-kubernetes-node-kubelet-serial which set PodPriority 

### LINKED ISSUE #81824: [Failing Tests] ci-kubernetes-node-kubelet-serial which set PodPriority are failing when adminssion Priority enable
Setting PodPriority directly when adminssion Priority enable,the Pod is rejected.
To fix this, we need to create new priority classes for these tests.

**Which jobs are failing**:

https://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-serial/

**Which test(s) are failing**:
_output/local/go/src/k8s.io/kubernetes/test/e2e_node/critical_pod_test.go:46
I0822 19:01:41.165]     should be able to create and delete a critical pod [It]

**Since when has it been failing**:

**Testgrid link**:
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-serial/1164593577464958978

**Reason for failure**:
because adminssion Priority enable，If the priority class is not found, the Pod is rejected.

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by zouyee**
**What type of PR is this?**
/kind failing-test

**What this PR does / why we need it**:
Setting PodPriority directly when adminssion Priority enable,the Pod is rejected.
To fix this, we need to create new priority classes for these tests.

**Which issue(s) this PR fixes**:

Fixes #81824

**Special notes for your reviewer**:

Looks like adminssion `Priority plugin` enable was added to prevent pod created:
To fix this, we need to create new priority classes for these tests.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by zouyee** [comment]
@oomichi  PTAL

File: test/e2e_node/critical_pod_test.go [hunk 1]
```diff
@@ -56,6 +58,8 @@ var _ = framework.KubeDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:C
 			if !configEnabled {
 				framework.Skipf("unable to run test without dynamic kubelet config enabled.")
 			}
+			// because adminssion Priority enable，If the priority class is not found, the Pod is rejected.
```
Comment (neolit123) [inline test/e2e_node/critical_pod_test.go]: the `，` here is a https://www.fileformat.info/info/unicode/char/ff0c/index.htm
please use `,` instead.

File: test/e2e_node/critical_pod_test.go [hunk 2]
```diff
@@ -67,20 +71,28 @@ var _ = framework.KubeDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:C
...
+			}, nil)
+			nonCriticalBestEffort := getTestPod(false, bestEffortPodName, v1.ResourceRequirements{}, nil)
 			criticalPod := getTestPod(true, criticalPodName, v1.ResourceRequirements{
 				// request the entire resource capacity of the node, so that
 				// admitting this pod requires the other pod to be preempted
 				Requests: getNodeCPUAndMemoryCapacity(f),
-			})
+			}, systemCriticalPriority)
 
+			ginkgo.BeforeEach(func() {
```
[the remark this answers is not in the document]
Comment (zouyee) [reply test/e2e_node/critical_pod_test.go]: got it

**Review by oomichi — COMMENTED** [review]
Thanks for updating, I forgot to point this in my previous review. Sorry about that.

File: test/e2e_node/critical_pod_test.go [hunk 3]
```diff
@@ -67,19 +71,22 @@ var _ = framework.KubeDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:C
...
+			nonCriticalBestEffort := getTestPod(false, bestEffortPodName, v1.ResourceRequirements{}, nil)
 			criticalPod := getTestPod(true, criticalPodName, v1.ResourceRequirements{
 				// request the entire resource capacity of the node, so that
 				// admitting this pod requires the other pod to be preempted
 				Requests: getNodeCPUAndMemoryCapacity(f),
-			})
+			}, systemCriticalPriority)
+
+			_, err = f.ClientSet.SchedulingV1().PriorityClasses().Create(systemCriticalPriority)
+			gomega.Expect(err == nil || errors.IsAlreadyExists(err)).To(gomega.BeTrue())
```
[the remark this answers is not in the document]
Comment (zouyee) [reply test/e2e_node/critical_pod_test.go]: Thank you for your detailed explanation.

File: test/e2e_node/critical_pod_test.go [hunk 4]
```diff
@@ -122,7 +131,7 @@ func getNodeCPUAndMemoryCapacity(f *framework.Framework) v1.ResourceList {
 	}
 }
 
-func getTestPod(critical bool, name string, resources v1.ResourceRequirements) *v1.Pod {
+func getTestPod(critical bool, name string, resources v1.ResourceRequirements, scp *schedulingv1.PriorityClass) *v1.Pod {
```
Comment (dashpole) [inline test/e2e_node/critical_pod_test.go]: I don't think we need to pass this priorityclass in.  Just set it in the pod based on the `critical` bool.

Comment (oomichi) [reply test/e2e_node/critical_pod_test.go]: +1 if we can name `pod.Spec.PriorityClassName` without `f := framework.NewDefaultFramework("critical-pod-test")` which is defined outside of this function.

**Comment by yujuhong** [comment]
/approve

Setting the milestone since this fixes a failing test

**Pull request merged**